pysph/examples/rigid_body/rigid_block_in_tank.py

```diff
@@ -1,191 +1,250 @@
-"""A block of rigid solid falling on a tank of water. (6 minutes)
+"""A block of rigid solid falling on wood in a tank of water.
 
 This also demonstrates that rigid-rigid collisions work as well.
 
 """
-# NumPy
+from __future__ import print_function
 import numpy as np
+import matplotlib.pyplot as plt
```

Comment:
The test builders may not have matplotlib so it is better to have this import inside the post_process method.